packages/eslint-config-next/README.md

```diff
@@ -8,10 +8,10 @@ Kilo.Health ESLint config for Next.js.
 
 ```bash
 # yarn
-yarn add -D eslint @babel/core
+yarn add -D eslint prettier @babel/core
 
 # npm
```

Comment:
Why do we need `prettier`?

Reply:
because of eslint-plugin-prettier. It has prettier as peer dependency